lighthouse-core/test/index-test.js

```diff
@@ -87,7 +87,7 @@ describe('Module Tests', function() {
       });
   });
 
-  it('should return formatted audit results when given no categories', function() {
```

Comment:
oh wow you don't even get any category results? yeah let's nuke this

Reply:
added a new test to assert things finish even if no categories in the config